conf/defaults.ini

```diff
@@ -155,6 +155,9 @@ google_tag_manager_id =
 
 #################################### Security ############################
 [security]
+# disable creation of admin user on first start of grafana
+disable_admin_user = false
```

Comment:
I would prefer a name that is a bit more clear, disable_initial_admin_creation , or something that make it clear that this not disabling any user but the creation of the first default admin user

Reply:
Good point